tests/AppBundle/Controller/BlogControllerTest.php

```diff
@@ -54,4 +55,38 @@ public function testRss()
             'The xml file displays the right number of posts.'
         );
     }
+
+    /**
+     * This test changes the database contents by creating a new comment. However,
+     * thanks to the DAMADoctrineTestBundle and its PHPUnit listener, all changes
+     * to the database are rolled back when this test completes. This means that
+     * all the application tests begin with the same database contents.
+     */
+    public function testNewComment()
+    {
+        $client = self::createClient([], [
+            'PHP_AUTH_USER' => 'john_user',
+            'PHP_AUTH_PW' => 'kitten',
+        ]);
+
+        /** @var Post $post */
+        $post = $client->getContainer()->get('doctrine')->getRepository(Post::class)->find(1);
```

Comment:
@yceruto, Sorry for confusing you! I mixed up the `doctrine` service and an entity manager. :(